### DISCUSSION TITLE
Is align-self: end supported?

### DISCUSSION THREAD

**Discussion by NetFlareSuperAdmin** [tailwindlabs/tailwindcss#11717] [Help]
I know `self-end` is for `align-self: flex-end`.

but what about `align-self: end`? Any class for it?

**Comment by wongjn** [comment]
`flex-end` and `end` are synonymous, as per [the CSS specification](https://www.w3.org/TR/css-align-3/#valdef-self-position-flex-end):

> When used outside of a [flex formatting context](https://www.w3.org/TR/css-flexbox-1/#flex-formatting-context), this value behaves as [`end`](https://www.w3.org/TR/css-align-3/#valdef-self-position-end).

Hence, you can use `self-end` where you would like to apply `align-self: end` or `align-self: flex-end`.
